pyart/aux_io/arm_vpt.py

```diff
@@ -218,3 +218,269 @@ def read_kazr(
         elevation,
         instrument_parameters=instrument_parameters,
     )
+
+
+def read_mmcr(
+    filename,
+    field_names=None,
+    additional_metadata=None,
+    file_field_names=False,
+    exclude_fields=None,
+    include_fields=None,
+    mode_names=None,
+    mode_to_extract=None,
+):
+    """
+    Read millimeter wavelength cloud radar (MMCR) b1 NetCDF ingest data.
+
+    Parameters
+    ----------
+    filename : str
+        Name of NetCDF file to read data from.
+    field_names : dict, optional
+        Dictionary mapping field names in the file names to radar field names.
+        Unlike other read functions, fields not in this dictionary or having a
+        value of None are still included in the radar.fields dictionary, to
+        exclude them use the `exclude_fields` parameter. Fields which are
+        mapped by this dictionary will be renamed from key to value.
+    additional_metadata : dict of dicts, optional
```

Comment:
I would be fine with removing this here and in the function above and just setting the call below to None if not used.

Reply:
Yeah we can remove both in a future PR.